tests/ui/traits/solver-cycles/129541-recursive-struct-and-array-impl.rs

```diff
@@ -0,0 +1,23 @@
+// Regression test for #129541
+
+//@ error-pattern: reached the recursion limit finding the struct tail for `<[Hello] as Normalize>::Assoc`
+
+trait Bound {}
```

Comment:
why add this test?

Reply:
This was just necessary before I added the normalization.